src/renderer/views/Watch/Watch.js

```diff
@@ -163,6 +163,9 @@ export default Vue.extend({
     },
     theatrePossible: function () {
       return !this.hideRecommendedVideos || (!this.hideLiveChat && this.isLive) || this.watchingPlaylist
+    },
+    currentLocale: function () {
+      return this.$store.getters.getCurrentLocale
```

Comment:
Please don't use `getCurrentLocale`, instead do it the same way that is done in https://github.com/FreeTubeApp/FreeTube/pull/2557.

Reply:
This should be fixed now.
You were right that I didn't need to specify 'default' when doing it this way.